save_embed_elmo.py

```diff
@@ -0,0 +1,69 @@
+import multiprocessing as mp
+import pickle as pk
+import sys
+#import h5py
+#pip install absl-py
+#pip install tensorflow
+#pip install tensorflow_hub
+#pip install tqdm
+import tensorflow as tf
+import tensorflow_hub as hub
+#from sentence_transformers import SentenceTransformer
+import numpy as np
+import pandas as pd
+from tqdm import tqdm
+
+_NUM_THREAD = (mp.cpu_count() - 1)
+
+tf.compat.v1.disable_eager_execution()
+model_weights_path = '../../data_files/elmo_3'
+closed_d3m_file = '../../data_files/data/closed_d3m_data.csv'
+file = '../data_files/data/sample.csv'
+open_d3m_file = '../../data_files/data/open_d3m_data.csv'
+
+def initialize_model(model_path):
+    model = hub.Module(model_path)
+    return model
+    
+def embed(data, model_weights_path):
```

Comment:
this looks like more duplicate code, but a different language model. If we are playing with multiple language models, let's make that configurable, instead of duplicating code.